src/Simplex/Chat/Controller.hs

```diff
@@ -448,6 +448,14 @@ data ChatCommand
   | APISetConnectionIncognito Int64 IncognitoEnabled
   | APIChangeConnectionUser Int64 UserId -- new user id to switch connection to
   | APIConnectPlan UserId AConnectionLink
+  | APIPrepareInvitationContact UserId InvitationLinkData ACreatedConnLink
+  | APIPrepareAddressContact UserId AddressLinkData ACreatedConnLink
```

Comment:
looking at the types, it seems like invitation/address link data, and all relevant APIs can be merged into one, and the rest managed via contact status.

Otherwise it creates too much taxonomy that would be largely duplicated.

Welcome message would still remain Maybe, and we would still allow sending only one message prior to connecting... Maybe we do need to have some additional flags though to differentiate.